### PR TITLE
HPE Switch Modeling

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-10203](https://jira-dc.paloaltonetworks.com/browse/CIAC-10203)

## Description
This PR contains a new content pack for modeling HPE Switches event logs. 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/HPESwitch/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# HPE Switch
+<~XSIAM>
+This pack includes Cortex XSIAM content. 
+
+## Configuration on HPE Switches
+This section describes the basic mandatory steps you should perform on an HPE switch device in order to forward its audited logs to XSIAM via Syslog.
```
Comment (ShirleyDenkberg) [inline Packs/HPESwitch/README.md:6]:
```suggestion
This section describes the basic mandatory steps you should perform on an HPE switch device in order to forward its audited logs to Cortex XSIAM via Syslog. 
```

File: Packs/HPESwitch/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# HPE Switch
+<~XSIAM>
+This pack includes Cortex XSIAM content. 
+
+## Configuration on HPE Switches
+This section describes the basic mandatory steps you should perform on an HPE switch device in order to forward its audited logs to XSIAM via Syslog. 
+
+### Configure Syslog forwarding
+HPE's switches supports forwarding the audited messages to a remote Syslog server. This is done via the ***[logging](https://www.arubanetworks.com/techdocs/AOS-CX/10.14/HTML/diagnostics_8100-83xx-9300-10000/Content/Chp_RSyslog/RSyslog_cmds/log-10.htm)*** command.
```
Comment (ShirleyDenkberg) [inline Packs/HPESwitch/README.md:9]:
```suggestion
HPE's switches support forwarding the audited messages to a remote Syslog server. This is done via the ***[logging](https://www.arubanetworks.com/techdocs/AOS-CX/10.14/HTML/diagnostics_8100-83xx-9300-10000/Content/Chp_RSyslog/RSyslog_cmds/log-10.htm)*** command. 
```

File: Packs/HPESwitch/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+<~XSIAM>
+This pack includes Cortex XSIAM content. 
+
+## Configuration on HPE Switches
+This section describes the basic mandatory steps you should perform on an HPE switch device in order to forward its audited logs to XSIAM via Syslog. 
+
+### Configure Syslog forwarding
+HPE's switches supports forwarding the audited messages to a remote Syslog server. This is done via the ***[logging](https://www.arubanetworks.com/techdocs/AOS-CX/10.14/HTML/diagnostics_8100-83xx-9300-10000/Content/Chp_RSyslog/RSyslog_cmds/log-10.htm)*** command. 
+
+Follow these steps to configure forwarding of event logs from an HPE switch to an XSIAM Syslog Broker VM over UDP:
```
Comment (ShirleyDenkberg) [inline Packs/HPESwitch/README.md:11]:
```suggestion
Follow these steps to configure forwarding of event logs from an HPE switch to a Cortex XSIAM Syslog Broker VM over UDP:
```

File: Packs/HPESwitch/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+1. Connect to the switch CLI (Command Line Interface). 
+2. Type **enable** to move from the _Operator Level_ mode to  _Manager Level_ mode, followed by the *Manager Level* password if prompted. 
+3. Type **config** to enter the _Global Configuration_ command mode. 
+4. Type **logging**  **_\<IP\>_** **udp** **_\<PORT\>_** where _\<IP\>_ and _\<PORT\>_ are the corresponding IP address and port of the target [XSIAM Syslog Broker VM](#broker-vm).
```
Comment (ShirleyDenkberg) [inline Packs/HPESwitch/README.md:15]:
```suggestion
4. Type **logging**  **_\<IP\>_** **udp** **_\<PORT\>_** where _\<IP\>_ and _\<PORT\>_ are the corresponding IP address and port of the target [Cortex XSIAM Syslog Broker VM](#broker-vm). 
```

File: Packs/HPESwitch/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+   Password:
+   HP Switch# configure
+   HP Switch(config)# logging 192.168.1.10 udp 514
+   switch(config)# write memory
+   switch(config)# **exit**
+   switch# exit
+```
+
+#### Remark 
+For additional examples and command options, such as setting the logging severity level, filtering logging only for certain event id's and forwarding syslog messages over TCP or TLS, checkout HPE's [Remote Syslog logging command reference](https://www.arubanetworks.com/techdocs/AOS-CX/10.14/HTML/diagnostics_8100-83xx-9300-10000/Content/Chp_RSyslog/RSyslog_cmds/rem-sys-com.htm).
```
Comment (ShirleyDenkberg) [inline Packs/HPESwitch/README.md:33]:
```suggestion
For additional examples and command options, such as setting the logging severity level, filtering logging only for certain event ID's and forwarding syslog messages over TCP or TLS, see HPE's [Remote Syslog logging command reference](https://www.arubanetworks.com/techdocs/AOS-CX/10.14/HTML/diagnostics_8100-83xx-9300-10000/Content/Chp_RSyslog/RSyslog_cmds/rem-sys-com.htm).
```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

**Review by OBavly — APPROVED** [review]

**Pull request merged**